pages/index.tsx

```diff
@@ -7,6 +7,23 @@ export const Home = (): JSX.Element => (
     <Head>
       <title>First Contributions</title>
       <link rel="icon" href="/favicon.ico" />
+      <script
+        dangerouslySetInnerHTML={{
+        __html: `
+  var _paq = window._paq = window._paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//webstats.gnome.org/";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', '31']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+  })();
+        `
+      }}>
+</script>
```

Comment:
We cannot accept this PR, You can't add custom trackers into the project. @Roshanjossey please note this PR

Reply:
I think this user is just spamming. I saw 2 PRs in backend repo too
https://github.com/firstcontributions/backend/pull/31
https://github.com/firstcontributions/backend/pull/30

Closing this